### PR TITLE
Sudoku solver

### PR DESCRIPTION
**Pull request by yashmita**
Added code for Sudoku Solver in Java for issue #42 
Kindly review it.

### CODE REVIEW

File: Code/Java/Sudoku_Solver.java [hunk 1]
```diff
@@ -0,0 +1,149 @@
+/*
+PROBLEM STATEMENT:
+Given an nxn sized incomplete Sudoku in which 0 represents an empty cell. The task is to print it's solution.  
+*/
+
+import java.util.Scanner;
+class Sudoku_Solver
+{
+	public static boolean CanPlace(int[][] mat, int n, int i, int j, int num)
```
Comment (raksha009) [inline Code/Java/Sudoku_Solver.java:9]: Plz add comments above each func, denoting what it does.

File: Code/Java/Sudoku_Solver.java [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+9 0 0 8 6 3 0 0 5 
+0 5 0 0 9 0 6 0 0 
+1 3 0 0 0 0 2 5 0
+0 0 0 0 0 0 0 7 4 
+0 0 5 2 0 6 3 0 0
+Output:
+No Solution!
+
+TIME COMPLEXITY:  O(9^(n*n)), as there will be 9 options from 1 to 9 to fill in an empty cell
+SPACE COMPLEXITY: O(n*n)
```
Comment (raksha009) [inline Code/Java/Sudoku_Solver.java:148]: Specify n

**Review by raksha009 — CHANGES_REQUESTED** [review]

**Comment by yashmita** [comment]
Okay, I've updated the changes as suggested. Kindly have a look.

**Review by raksha009 — APPROVED** [review]
LGTM 👍🏼

**Pull request merged**